cucumber/features/careerPage.feature

```diff
@@ -2,3 +2,30 @@ Feature: EPAM job searching
   As a Job searcher
   I want to browser through EPAM Job offers by various criteria
   So I can find to best fitting offer for me
+
+Scenario Outline: Search for <PositionName> in <City>
```

Comment:
Could you please add a number for the scenario. It would be great, if we can distinquish the to examples with e.g: "a", "b" marker.

Reply:
done